services/bots.py

```diff
@@ -117,10 +117,28 @@ def get_token_type_mapping(repo: Repository):
             "Repository has no good bot for admin, but still continuing operations in case it is not doing an admin call anyway",
             extra=dict(repoid=repo.repoid),
         )
+
+    if get_config("github", "integration", "id") and get_config(
```

Comment:
I think you need a "if service is github" here otherwise it will try to do this for bb/gl